src/components/Message/Message.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react';
+import cx from 'classnames';
+
+import styles from './Message.module.scss';
+
+interface MessageProps {
+  children?: ReactNode;
+  heading?: string;
+  text: string;
+  kind?: 'error' | 'info';
```

Comment:
> Out of curiosity, why do you use kind instead of common type?

It just felt right. ;) I guess I just mentally associate the "type" property too much with buttons.